.ebextensions/yarn.config

```diff
@@ -0,0 +1,7 @@
+commands:
+  upgrade_nodejs_and_install_yarn:
+    command: |
+      /usr/bin/curl -sLO https://rpm.nodesource.com/pub_10.x/el/7/x86_64/nodesource-release-el7-1.noarch.rpm
```

Comment:
Is this definitely idempotent? It’ll get run on every deploy.

Reply:
No. I added `test` to check if version 10.x of node is installed.

Reply:
Cool. You might also be able to do something with the yum support in ebextensions to make it a little simpler.

Reply:
https://docs.aws.amazon.com/elasticbeanstalk/latest/dg/customize-containers-ec2.html#linux-packages has some examples.